### PR TITLE
Add artefact- System Design

### PR DESCRIPTION
**Pull request by udhaymanhas**
Added system design pdf, updated readme

### CODE REVIEW

**Comment by chirgjn** [comment]
Please incorporate the feedback from #8

**Review by chirgjn — CHANGES_REQUESTED** [review]
Please incorporate the feedback from #8

**Comment by chirgjn** [comment]
Please verify changes and incorporate previously recommended changes

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # Juntos
 Juntos is a web application that lets you co watch streaming service such as youtube. More services to be added in future such as Netflix, Prime, HBO NOW ...etc.
+
+# Artefacts
+## <a href="prd/Juntos System Design.pdf">System Design</a>
```
Comment (chirgjn) [inline README.md:5]: > Please verify the changes.
The link isn't valid.

**Review by chirgjn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (udhaymanhas) [reply README.md]: resolved the link issue, changed file name and used markdown link syntax

**Review by chirgjn — CHANGES_REQUESTED** [review]
Please refer to review comments in #8

**Review by chirgjn — COMMENTED** [review]
Please resolve conflicts

File: README.md [hunk 2]
```diff
@@ -3,3 +3,7 @@ Juntos is a web application that lets you co watch streaming service such as you
 
 # Artefacts  
 * [PRD](/prd/Juntos_PRD.pdf) 
+* [System Design](/system-design/Juntos_system_design.pdf)
+
+
+
```
Comment (chirgjn) [inline README.md:9]: Nit: Please remove extra whitespace

**Review by chirgjn — COMMENTED** [review]
It's preferred to keep filenames in lowercase unless required otherwise

File: README.md (hunk 2 above)
Comment (udhaymanhas) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -2,4 +2,5 @@
 Juntos is a web application that lets you co watch streaming service such as youtube. More services to be added in future such as Netflix, Prime, HBO NOW ...etc.
 
 # Artefacts  
-* [PRD](/prd/Juntos_PRD.pdf) 
+* [Product Requirement Document](/prd/Juntos_PRD.pdf)
```
Comment (chirgjn) [inline README.md:5]: It's preferred to keep the commits in the PR limited to only contain relevant changes

**Review by chirgjn — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (udhaymanhas) [reply README.md:5]: I thought you want me to change the name of the PRD. It is supposed to be capital only, so instead, I changed the entire name. You want me to revert it?

Comment (chirgjn) [reply README.md:5]: No need to revert this is more of a FYI comment.

The 'lowercase' comment was for the "filename" i.e. "juntos_system_design.pdf" is preferred over "Juntos_system_design.pdf".

Comment (udhaymanhas) [reply README.md:5]: oh! I will take care of this moving forward

**Pull request merged**